pyproject.toml

```
@@ -49,6 +49,11 @@ statsd = "^4.0.1"
 pymemcache = "^4.0.0"
 defusedxml = "^0.7.1"
 
+opentelemetry-sdk = "^1.20.0"
+opentelemetry-instrumentation-flask = "^0.41b0"
+opentelemetry-instrumentation-requests = "^0.41b0"
+opentelemetry-exporter-otlp-proto-http = "^1.20.0"
```

Better to bump these to the latest versions and run `poetry lock` since you will need to rebase your code anyway:

```suggestion
opentelemetry-sdk = "^1.22.0"
opentelemetry-instrumentation-flask = "^0.43b0"
opentelemetry-instrumentation-requests = "^0.43b0"
opentelemetry-exporter-otlp-proto-http = "^1.22.0"
```

---

Bumped